### ISSUE TITLE
New CCminers Needed

### ISSUE THREAD

**Issue by MaynardMiner** [MaynardMiner/MM.Hash#3]
If anyone uses miner, and sees a better CCMiner to be used for certain algos- Please let me know. I will add to application immediately.

Miner must be able to be built and compiled, ideally under the ./build.sh command. Not many developers are still supporting linux compiling.

**Comment by MaynardMiner** [comment]
Actually, I'm going to change program to download git, which hopefully should resolve this

**Comment by slovke** [comment]
I can send you tonight miners screen for 1080Ti cards for all algos which they work best on 1080Ti`s only couse i use them and benchmarked them

**Comment by MaynardMiner** [comment]
I'm rewriting the entirety of the cloning process. Before it was downloading released linux zips, and then extracting them.

I just figured out how to rewrite now so that it will clone the git directly. This means any miner on github can now be used, not the ones that made linux releases...So long as they left the build.sh file in CCminer.

**Comment by slovke** [comment]
![image](https://user-images.githubusercontent.com/21175543/39678565-48c3f16e-518f-11e8-9f27-db873b13b48e.png)
this is best miner for given algos for 1080TI in my rigs. Hope its of some help to you :)

**Comment by MaynardMiner** [comment]
KlausT & Alexis will not compile correctly using the conventional Autogen.sh Configure and Build.sh method in my miner. Going to see if there is another way to make it work. Downloading the .zip source and compiling failed as well. I may need to see if they rely on older dependencies...And that may be the case, but working on it.

Confirmed Dual-Claymore works, however there is no downloading source (They do not release it). It will come pre-built in next release once current bugs are solved. Will look into dtsm, because I actually liked using it back in the day.

Will see if HcashOrg CCminer works (HSR). 

Not going to do nanashi- It's two years old- no updates or support.

**Comment by MaynardMiner** [comment]
To resolve this issue- I decided to do expirementing, and came up with this-

If you notice most of your CCMiners are probably all old...Like built with Cuda 7.5.

They were likely all written a year or two ago, and people have just been using them because they still work, even though they were originally compiled with what is now dinosaur technology.

However, if you tried to compiled them now- A lot of them are outdated and won't compile with latest toolkit.

I figured out how to get all the CCminers (not one makred expiremental) in MM.Hash to complie under 9.1, which support recent your nvidia card's recent drivers, and yield slightly better performamce. I'm working on getting Lennis's Ccminer, and probably Nanashi, too. It takes awhile to pick apart their miners to see what they are missing.

I decided however, since these devs are lazy and either won't update their ccminer's source for linux (or windows in some cases)- I'm going to compile them with a 1% dev fee that is split between me and original creator for playing Frankenstein and breathing life into their outdated work. Right now, being in beta- They are free to use. I am only doing this for miners that won't compile under linux cuda 9.1.

I never noticed that I've been using such outdated miners until I switched to linux.

Closing because I know I have to trial and error miners to Frankenstein them to work.
